base/src/main/java/org/aya/concrete/desugar/BinExprParser.java

```diff
@@ -57,7 +56,7 @@ public BinExprParser(@NotNull AyaBinOpSet opSet, @NotNull SeqView<@NotNull Elem<
     throw new IllegalArgumentException("not an operator");
   }
 
-  @Override protected @Nullable OpDecl asOpDecl(@NotNull Elem<Expr> elem) {
+  @Override protected @Nullable OpDecl asOpDecl(@NotNull Expr.NamedArg elem) {
```

Comment:
Suggest rename this method to: `referredOpDecl`

Reply:
I like `underlyingOpDecl`